lambda-classes.js

```diff
@@ -0,0 +1,107 @@
+// // CODE here for your Lambda Classes
+
+class Person {

```

Comment:
Awesome job on this one, you could try a higher order function to get the arrays to print out as strings instead of an array full of strings.